src/sidebar/directive/test/h-branding-fixtures.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+// Test data for the firehose of branding combinations
+module.exports = [
+
+  // ALL SUPPORTED PROPERTIES
+  {
+    settings: {appBackgroundColor: 'blue'},
+    attrs: 'h-branding="appBackgroundColor"',
+    styleChanged: 'backgroundColor',
```

Comment:
I think it is helpful to indicate what are the inputs vs. expected outputs in fixtures. One pattern we have in some of our tests is to use an "expected" prefix on properties that define outputs. See https://github.com/hypothesis/client/blob/master/src/sidebar/test/virtual-thread-list-test.js#L100 for example.